etc/env/example.env

```diff
@@ -2,13 +2,20 @@
 # - - -
 
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization jupyter
+CANDIG_MODULES=tyk keycloak vault weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization
 # options are [<ip_addr>, <url>]
 CANDIG_DOMAIN=candig.local
+CANDIG_AUTH_DOMAIN=keycloak
+#candig-auth.local
+
+# candig server
+CANDIG_SERVER_VERSION=1.3.0
+CANDIG_SERVER_HOST=0.0.0.0
+CANDIG_SERVER_PORT=3001
 
 # miniconda venv
 # options are [linux, darwin]
-VENV_OS=linux
+VENV_OS=darwin
```

Comment:
linux should be the default as it is more likely to be the default deploy environment than OS X